### ISSUE TITLE
Twenty Twenty: Up Jump Link Doesn't Work for Screen Readers

### ISSUE THREAD

**Issue by donalirl** [Automattic/themes#1999]
## Steps to replicate

1. Using a screen reader such as Apple Voiceover, navigate to the bottom of the page of a site with the Twenty Twenty theme
2. Select the Up jump link that is used to send you back to the top
3. Notice this does not work with a screen reader.

This worked fine on the .org demo site, but not on the .com demo site or the user-reported site.

## Screencast: 
(make sure to enable audio on the video player)

https://user-images.githubusercontent.com/30727666/128828245-a5a0dab8-687a-44cf-a911-a17bf84eec0e.mp4

## Seen at:

Chrome (latest) on Samsung Galaxy S10e with android voice assists nt
Safari (latest) on iPod touch with Apple Voiceover

## Expected

Using a screen reader, I expected to be able to jump to the top using the Up link.

* User report: #2889143-zen #p4vt7e-6n-p2

#accessibility

**Comment by mrfoxtalbot** [comment]
I did a quick test and this appears to be working correctly. I tried a [Google Chrome Screen Reader Extension](https://chrome.google.com/webstore/detail/screen-reader/kgejglhpjiefppelpmljglcjbhoiplfn?hl=en), but I am not sure about other screen readers.

https://user-images.githubusercontent.com/4452464/175518673-35ca52c3-e326-405f-b887-364fd9b4b2ac.mp4

 I am going to mark it as solved but if you can still reproduce it @donalirl , please feel free to reopen it.
